src/horust/supervisor/process_spawner.rs

```diff
@@ -169,14 +170,26 @@ fn redirect_output(
     Ok(())
 }
 
+/// Find program on PATH.
+///
+fn find_program_path(program_name: &String) -> std::result::Result<String, Errno> {
+    for dir in std::env::var("PATH").unwrap().split(":") {
```

Comment:
You can use https://doc.rust-lang.org/std/env/fn.split_paths.html. Please don't unwrap but rather use `.ok_or(Errno::NotFound)?;`